desktop-widgets/statswidget.cpp

```diff
@@ -93,6 +93,8 @@ StatsWidget::StatsWidget(QWidget *parent) : QWidget(parent)
 	view = qobject_cast<StatsView *>(root);
 	if (!view)
 		qWarning("Oops. The root of the StatsView is not a StatsView.");
+	if (view)
+		view->setVisible(isVisible()); // Synchronize visibility of widget and QtQuick-view.
```

Comment:
ahh, that comment makes more sense. Thanks - didn't realize you had pushed a new version